src/Views/Wrappers/ViewWrapper.vala

```diff
@@ -371,7 +371,6 @@ public abstract class Noise.ViewWrapper : Gtk.Grid {
             set_active_view (ViewType.GRID);
         else if (has_list_view) {
             view_stack.visible_child = list_view;
-            list_view.list_view.scroll_to_current_media (true);
```

Comment:
Is there a reason this is removed? I think we did this so that we always keep the currently playing song in view

Reply:
I've removed this because if you delete a song from the queue it will jump to the currently played song which I don't think is the correct behavior if i want to edit the queue (will you can try both to see which is better :D)